gp-advanced-save-and-continue/gpasc-use-dropdown-to-toggle-enabled.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Gravity Perks // Advanced Save and Continue // Toggle GPASC with a Dropdown
+ * https://gravitywiz.com/documentation/gravity-forms-advanced-save-continue/
+ *
+ * This snippet allows you to toggle GPASC client side with a dropdown.
```

Comment:
I forgot... Can you add an `Instructions` block here?

I typically go with something like...

```
 * Instructions:
 *  1. Add snippet to form using https://gravitywiz.com/gravity-forms-custom-javascript/
 *  2. Customize X, Y, and Z accordingly
```

Reply:
Done! (finally lmaoooo)